build.sh

```diff
@@ -35,7 +35,7 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$1}" )
             args=( "${args[@]/$2}" )
             shift
-            ;;        
+            ;;
```

Comment:
@333fred I keep seeing unrelated whitespace cleanup. Is your editor configured to do that? It's not code review friendly.